### PR TITLE
CIF-996 - Show the store-related properties in the cloud bound folder…

### PR DESCRIPTION
**Pull request by LSantha**
With the multi-store bindings in place we also need to show the store-related properties in the cloud bound folder.

## Description

Include the custom fields of the bindproducttreewizard in the properties page of ccifrootfolder.

## Related Issue

CIF-996

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [X] All unit tests pass on CircleCi.
- [X] I ran all tests locally and they pass.

### CODE REVIEW

File: content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/ccifrootfolder/_cq_dialog/.content.xml [hunk 1]
```diff
@@ -55,6 +55,10 @@
                                             name="./cq:catalogIdentifier"
                                             renderReadOnly="{Boolean}true"
                                             required="{Boolean}false"/>
+                                    <customfields
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType="commerce/components/ccifrootfolder/include"
```
Comment (mhaack) [inline content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/ccifrootfolder/_cq_dialog/.content.xml:60]: We should not use "ccif" any more for new things we introduce. I know it is still there for old code, but please not for new.

Comment (LSantha) [reply content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/ccifrootfolder/_cq_dialog/.content.xml:60]: I agree with not using ccif. The situation here is that the new include component belongs to the ccifrootfolder and so putting it under ccifrootfolder looks like a logical choice.

How about renaming the two components starting with ccif to start with cif?

Comment (mhaack) [reply content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/ccifrootfolder/_cq_dialog/.content.xml:60]: > How about renaming the two components starting with ccif to start with cif?

+1 for clean that up if we touching that area anyway

Comment (LSantha) [reply content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/ccifrootfolder/_cq_dialog/.content.xml:60]: @mhaack, I had a quick look at the cleanup which will affect quite a bit more files and I think I would do that refactoring in a dedicated issue to keep the current feature-work clean, so I created CIF-1027.

Comment (mhaack) [reply content/cif-virtual-catalog/src/main/content/jcr_root/libs/commerce/components/ccifrootfolder/_cq_dialog/.content.xml:60]: Ok

**Review by mhaack — APPROVED** [review]

**Pull request merged**